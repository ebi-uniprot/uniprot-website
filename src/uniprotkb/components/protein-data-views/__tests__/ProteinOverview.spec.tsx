import React from 'react';

import renderWithRouter from '../../../../shared/__test-helpers__/RenderWithRouter';

import ProteinOverview from '../ProteinOverviewView';

import swissprotData from '../../__mocks__/swissprotEntry.json';

describe('ProteinOverview component', () => {
  test('should render', () => {
    const { asFragment } = renderWithRouter(
      <ProteinOverview data={swissprotData} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
