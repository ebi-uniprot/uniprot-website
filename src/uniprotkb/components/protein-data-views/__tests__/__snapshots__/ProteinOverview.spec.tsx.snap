// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProteinOverview component should render 1`] = `
<DocumentFragment>
  <section>
    Amyloid-beta precursor protein · 
    <a
      href="/taxonomy/9606"
    >
      Homo sapiens
    </a>
     · 
    <strong>
      Gene:
    </strong>
     APP (A4, AD1) · 770 amino-acids · 1: Evidence at protein level · 
    <span
      title="Annotation Score"
    >
      <span
        class="doughnut-chart--small colour-platinum"
      >
        <span
          class="doughnut-chart--small__left-wrap"
        >
          <span
            class="doughnut-chart--small__left-wrap__loader colour-sea-blue"
            style="transform: rotate(180deg);"
          />
        </span>
        <span
          class="doughnut-chart--small__right-wrap"
        >
          <span
            class="doughnut-chart--small__right-wrap__loader colour-sea-blue"
            style="transform: rotate(180deg);"
          />
        </span>
        <span
          class="doughnut-chart--small__inner-circle"
        >
          5/5
        </span>
      </span>
    </span>
  </section>
</DocumentFragment>
`;
